### PR TITLE
ENH: add metatags to index head

### LINKED ISSUE #16: Add catalog metadata for display purposes
in the `<meta>` tag of the `index.html` file

Display name is already included, we also need more info in the correct tags, e.g. a description of the catalog and could an image be linked?

### PR DESCRIPTION
**Pull request by jsheunis**
This PR closes https://github.com/psychoinformatics-de/sfb1451-projects-catalog/issues/16

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -6,12 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
+    <title>CRC1451 Catalog</title>
+    <meta name="description" content="The CRC1451 data catalog for the German Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
+    <meta property="og:url" content="https://developer.mozilla.org"/>
```
Comment (jsheunis) [inline docs/index.html]:
```suggestion
    <meta property="og:url" content="https://psychoinformatics-de.github.io/sfb1451-projects-catalog"/>
```

File: docs/index.html [hunk 2]
```diff
@@ -6,12 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
+    <title>CRC1451 Catalog</title>
+    <meta name="description" content="The CRC1451 data catalog for the German Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```
Comment (mslw) [inline docs/index.html]:
```suggestion
    <meta name="description" content="The CRC1451 data catalog for the Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```
Probably doesn't need to say German - we are funded by DFG but we have international collaborations as well.

**Review by mslw — COMMENTED** [review]
All good, thanks. I left a suggestion for a technicality.

File: docs/index.html [hunk 3]
```diff
@@ -6,12 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
+    <title>CRC1451 Catalog</title>
+    <meta name="description" content="The CRC1451 data catalog for the Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
+    <meta property="og:url" content="https://psychoinformatics-de.github.io/sfb1451-projects-catalog"/>
+    <meta property="og:title" content="CRC1451 Catalog"/>
+    <meta property="og:locale" content="en-US"/>
+    <meta property="og:description" content="The CRC1451 data catalog for the German Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```
Comment (jsheunis) [inline docs/index.html]:
```suggestion
    <meta property="og:description" content="The CRC1451 data catalog for the Collaborative Research Center (CRC/SFB 1451) investigating key mechanisms of motor control in health and disease"/>
```

**Pull request merged**